### DISCUSSION TITLE
Github Discussions Lists

### DISCUSSION THREAD

**Discussion by TrDA-hab** [arendst/Tasmota#10668] [General]
Where is the real rating?   This rating is fictitious ???
![666](https://user-images.githubusercontent.com/56403720/105539109-66738b00-5d05-11eb-89c3-4f9d9dbd9429.jpg)     
**... and no, I don’t panic, I’m just asking (!) and nothing more!**

**Comment by sfromis** [comment]
That's no rating, just the number of times a response from the mentioned member has been selected as "the answer". Easiest way of getting a number there is to answer simple questions ;-) 

I'm not aware of any "real rating", which would not be a great concept anyway.

**Comment by ascillato** [comment]
Hi, That list is made automatically by github ( is not in our control ) and is only for **_Q&A discussions_**. That is just a list of the people where their answers were selected to the questions done in **_Q&A discussions_**.

That is not a rating and it is just an useless list. I don't know why Github have done that. Well, **_Github Discussions_** is still in Beta and under development by Github.

Please, let's ask to remove that list in **_Github Discussions Feedback_** ( https://github.com/github/feedback/discussions )

**Reply by TrDA-hab** [reply]
Thanks for the link, I will definitely use it!

**Comment by TrDA-hab** [comment]
Thank you very much!
I was sure that this is a rating. I'm sorry that I was wrong.
**It was just a question and nothing more!**
You don't need to delete anything. Thanks again!

p.s. I'm tired of answering this question in the PM, and asked it here let everyone understand.
